lms/djangoapps/api_manager/users/views.py

```diff
@@ -17,6 +18,7 @@
 
 from courseware import grades, module_render
 from courseware.model_data import FieldDataCache
+from courseware.views import get_module_for_descriptor, save_child_position, get_current_child
```

Comment:
We've added these imports but I don't see new references to them further down in the file?